# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-pdfium.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-137.0.7151.55/third_party/pdfium/core/fxcodec/png/png_decoder.cpp.vanilla	2025-05-28 22:15:55.001105834 +0200
+++ chromium-137.0.7151.55/third_party/pdfium/core/fxcodec/png/png_decoder.cpp	2025-05-28 22:16:32.816701675 +0200
@@ -52,8 +52,8 @@
                              CFX_DIBAttribute* pAttribute) {
   if (pAttribute) {
 #if defined(PNG_pHYs_SUPPORTED)
-    pAttribute->xdpi_ = png_get_x_pixels_per_meter(png_ptr, info_ptr);
-    pAttribute->ydpi_ = png_get_y_pixels_per_meter(png_ptr, info_ptr);
+    pAttribute->x_dpi_ = png_get_x_pixels_per_meter(png_ptr, info_ptr);
+    pAttribute->y_dpi_ = png_get_y_pixels_per_meter(png_ptr, info_ptr);
     png_uint_32 res_x, res_y;
     int unit_type;
     png_get_pHYs(png_ptr, info_ptr, &res_x, &res_y, &unit_type);
